Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Confirm.hs

```diff
@@ -188,7 +193,7 @@ handler transporter req quote = do
                 riderName = req.mbRiderName,
                 transporter,
                 fromLocation = uBooking.fromLocation,
-                toLocation = uBooking.toLocation,
+                toLocation = Nothing,
```

Comment:
why this changed for `NormalBooking` case?

Reply:
will fix this